metricbeat/module/system/diskio/_meta/data.json

```diff
@@ -18,18 +18,49 @@
     "system": {
         "diskio": {
             "io": {
-                "time": 18566027
+                "time": 656
             },
-            "name": "disk0",
+            "iostat": {
+                "await": 0,
+                "busy": 0,
+                "queue": {
+                    "avg_size": 0
+                },
+                "read": {
+                    "await": 0,
+                    "per_sec": {
+                        "bytes": 0
+                    },
+                    "request": {
+                        "merges_per_sec": 0,
+                        "per_sec": 0
+                    }
+                },
+                "request": {
+                    "avg_size": 0
+                },
+                "service_time": 0,
+                "write": {
+                    "await": 0,
+                    "per_sec": {
+                        "bytes": 0
+                    },
+                    "request": {
+                        "merges_per_sec": 0,
+                        "per_sec": 0
+                    }
+                }
+            },
```

Comment:
This is weird, this reports always zero-values for me, I'll take a deeper look in a follow up.

Reply:
I have been trying this with a running metricbeat and it ends-up reporting non-zero values, so this seems ok and I only had bad luck here.